### PR TITLE
Added CONTRIBUTING.md (Issue#1287)

### PR DESCRIPTION
**Pull request by AbdulKabia**
This PR addresses [Issue#1287](https://github.com/redux-saga/redux-saga/issues/1287). I've added a `CONTRIBUTING.md`  to the repo. If there's any changes that need to be done in order to have this PR accepted, please let me know.

Kind regards,

Abdul

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Contributing to redux-saga
+Contributing to redux-saga isn't limited to just filing bugs, users are more than welcomed to make suggestions, report any issue they may find, and make pull requests to help better redux-saga.
```
Comment (Andarist) [inline CONTRIBUTING.md]: "to help better redux-saga." -> "to help make redux-saga better."

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Contributing to redux-saga
+Contributing to redux-saga isn't limited to just filing bugs, users are more than welcomed to make suggestions, report any issue they may find, and make pull requests to help better redux-saga.
+
+## Filing a bug
+When filing a bug, please follow the provided template:
+
+```
+### Description of the bug/issue
```
Comment (Andarist) [inline CONTRIBUTING.md]: Would be cool to mention that reproduction case helps a lot and if one is provided it's usually WAY quicker to fix the issue. Repros can be made as separate git repositories or created on http://codesandbox.io/

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+### Getting redux-saga
+1. Fork a copy of our repo
+2. Open up Git in an environment of your choice
+3. Run the following
+
+```
+$ git clone https://github.com/YOUR-GITHUB-PROFILE-NAME/redux-saga.git
+$ cd redux-saga
+$ npm install
```
Comment (Andarist) [inline CONTRIBUTING.md]: actually this should be `npm run bootstrap` now as we've started using lerna which manage this monorepo

**Comment by AbdulKabia** [comment]
Hmm, managed to update my entire fork and even the branch, but Travis is still failing to build it. All I did was change the .md file. Do you happen to have any idea as to why this build could be failing, @Andarist ?

**Comment by Andarist** [comment]
It's some installation issue, if it happens on master I'll investigate this later - but for now thanks for the great work!

**Pull request merged**